src/modules/showScores.js

```diff
@@ -0,0 +1,18 @@
+import Scores from './addScore.js';
+import rowDOM from './addDOMelement.js';
+
+const refresh = () => {
+  const buttonRefresh = document.querySelector('.table-container__button');
+  buttonRefresh.addEventListener('click', async () => {
+    const table = document.querySelector('.table-container__body');
+    // Resets inner HTML so that data won't be repeated
+
+    table.innerHTML = '';
+    const data = new Scores();
+    // Got array of data from API: scores and names.
+    const scoresAPI = await data.getScores();
+    rowDOM(table, scoresAPI);
+  });
```

Comment:
- [ ] Good job in implementing the changes so far, However, as requested by the Previous Reviewer, One of the key requirements for this milestone is that you follow the wireframe even though styling is not important. Please try to display the leaderboard scores correctly while following the wireframe design. Attached below is your current display, when I click on the refresh `Button` after saving the `Username` and `Score` nothing shows except the existing data that has already been saved. :slightly_smiling_face: 

![Dashborad](https://github.com/EvansSnave/leaderboard/assets/43528594/4f953f3c-9a27-414f-ac62-63b1b53a8b27)


![student fetch](https://github.com/EvansSnave/leaderboard/assets/43528594/c4f70dcb-c848-4921-9705-466c711d7e50)

Reply:
@Alusp I have been trying to solve this but it works fine for me. Can you send me a video when you click on the refresh button?

Reply:
@Alusp Hello? Can you explain to me in more detail the error? I don't get it. Can you send me the video? To solve this I need to know what is exactly happening